tests/console/validate_file_system.pm

```diff
@@ -54,6 +55,11 @@ sub run {
               sub { m/Version:\s+$luks_type.*/s };
         }
     }
+    if (defined $swap) {
```

Comment:
i have the impression this will be always true with `defined`, whatever the value of the $swap. i assume that you expect that this will defined in the yaml or not. Assuming that, this will work as you expected without problem. But maybe it would be more accurate to have something like `defined $swap && $swap`. Do i miss something?

Reply:
I was repeating the pattern above, but you're right, might confuse someone, then it is more simple just `if($swap)`.